appveyor.yml

```diff
@@ -0,0 +1,17 @@
+clone_folder: c:\gopath\src\github.com\alexflint\go-filemutex
+
+environment:
+  GOPATH: c:\gopath
+
+install:
+  - echo %PATH%
+  - echo %GOPATH%
+  - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
+  - go version
+  - go env
+
+build: off
```

Comment:
Do master only and enable on commit and pull request git hooks instead?

branches:
  only:
    - master

Reply:
interesting, why do you recommend that?  I prefer to have a CI result on the PR and to block PR merges on CI going green.